### PR TITLE
text2path wrapper for sdql2 as a shitty workaround to it not parsing paths

### PR DESCRIPTION
**Pull request by silicons**
Daily reminder I don't actually know how to modify sdql2 code and I just use it because somehow I get it to work?

### CODE REVIEW

File: code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm [hunk 1]
```diff
@@ -176,6 +176,12 @@
 /proc/_list_swap(list/L, Index1, Index2)
 	L.Swap(Index1, Index2)
 
+/proc/_text2path(text)
+	return text2path(text)
+
+/proc/_path2text(path)
+	return "[path]"
```
Comment (Cyberboss) [inline code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm]: This seems unecessary

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm (hunk 1 above)
Comment (kevinz000) [reply code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm]: Intentional to explicitly convert to string

**Comment by Cyberboss** [comment]
Name it something other than path2text then, because it does a lot more
than just that

**Comment by kevinz000** [comment]
done :D

**Comment by RemieRichards** [comment]
Fix it properly you bumhead.
Something to do with ```value()``` not handling paths, or ```object_type``` not handling paths PROPERLY, I would say.

**Comment by kevinz000** [comment]
right ill fix it later when i have the time/skill_to_read_sdql2_black_magic_code :'(

**Pull request closed without merging**